automation_tools/utils.py

```diff
@@ -96,6 +96,11 @@ def run_command(cmd=None):
         run(cmd)
 
 
+def version(satver='0'):
+    satver = str(satver)
+    return tuple(map(int, ('9999' if 'nightly' in satver else satver).split('.')))
```

Comment:
@lpramuk I see `upstream-nightly` instead of `nightly` in [satellite6-installer.yaml](https://github.com/SatelliteQE/robottelo-ci/blob/a1506356124395679fdb7093d995181a83903ba7/jobs/satellite6-installer.yaml#L46)

Reply:
What do we use `upstream-nightly` or `nightly` everywhere?

Reply:
There was downstream-nightly once and only for 6.2 (a.k.a pre-snap1 composes)
We could eventually `s/upstream-nightly/nightly` 

My code work for both:
```>>> satver='upstream-nightly'
>>> 'nightly' in satver
True
```